### DISCUSSION TITLE
Ideas for Madara telegram stickers

### DISCUSSION THREAD

**Discussion by AbdelStark** [keep-starknet-strange/madara#490] [Ideas]
Hey shinobis, 

We've created some Telegram stickers for Madara, and now we want to hear from you! We would love to hear your ideas for new, cool stickers that we can create.

You can submit your ideas here, in this discussion. Please be as descriptive as possible.

As an example, take a look at this sticker: 
![Karya_Seni_Tanpa_Judul-1](https://github.com/keep-starknet-strange/madara/assets/45264458/3c3a29d2-ef71-4b2a-be85-18a146fb77a5)

The description for this sticker was: 

```
"Madara is Blazingly Fast!"
   - Description: Show Madara in a sprint, leaving a trail of fire behind him. This represents the high speed and performance of the Madara project.
 ```
 
 We look forward to seeing your ideas!

**Comment by rkdud007** [comment]
"Give me First good issue"
- Description : Show Madara seems very hungry, ready to hunt, holding big sword. behind there are some other people also sparkling eyes in the darkness also ready to hunt

**Comment by chee-chyuan** [comment]
- Description: Madara using Kage Bunshin, signifying that each Madara instance can be easily instantiated

**Comment by rkdud007** [comment]
"Running my node on Sharingan"
- Discription : Show Madara have a sharingan eye, very focusing, ( show the right face) , and right side there is monitor of laptop inside there is madara node logs

**Comment by bajpai244** [comment]
"Review Please"

- Madara is sitting in front of a computer, waiting for a review of their PR.
